modules/local/filter_difftable.nf

```diff
@@ -2,10 +2,10 @@ process FILTER_DIFFTABLE {
 
     label 'process_single'
 
-    conda "conda-forge::gawk=5.1.0"
```

Comment:
Re-writing this process in Python is a quite a major change to smuggle into a PR of small fixes ;-).

Can you say why the process was re-written, and demonstrate that the output is the same relative to previous?

Reply:
If you prefer, I can also do the module change in a separate PR, that's not a problem!
Nope, the output changed; it was incorrect before, that's why I had to change it :') AWK suddenly filtered out all the genes and just produced an empty table (was not the case when I originally added the module).
I tried a lot to change it in AWK but it simply did not work and I have to be honest, I found the AWK code quite painful to deal with as I'm simply not experienced enough with it.

Reply:
Oh, but as a follow-up, I did check the output. The new module + [this code](https://github.com/nf-core/differentialabundance/pull/257/files#diff-93d05d71f7facfc0a15a9131995c3b783d0d6ac38ccfd0bf7b5e654410c5c7e3R403) makes it so that the results are the same as the DE genes according to the html report.

Reply:
Actually, could you raise this as a separate PR? I think that would make more sense and you can post the reasoning and the evidence there.

Reply:
Yes of course, see #264